tests/unit/forklift/test_legacy.py

```diff
@@ -1655,7 +1654,7 @@ def test_upload_fails_with_invalid_digest(self, pyramid_config, db_request,
 
         assert resp.status_code == 400
         assert resp.status == (
-            "400 The digest supplied does not match a digest calculated "
+            "400 The digest doesn't match a digest calculated "
```

Comment:
I think we are losing some context here about what the error is.

Reply:
Reverted.